tests/test_data/expected_excel_xtce.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version='1.0' encoding='utf-8'?>
+<xtce:SpaceSystem xmlns:xtce="http://www.omg.org/space/xtce" name="Test Instrument">
+	<xtce:Header date="2024-07-26 00:00:00" version="v1.2" author="Space Packet Parser" />
+	<xtce:TelemetryMetaData>
+		<xtce:ParameterTypeSet>
+			<xtce:IntegerParameterType name="VERSION" signed="false">
+				<xtce:IntegerDataEncoding sizeInBits="3" encoding="unsigned" />
+			</xtce:IntegerParameterType>
+			<xtce:IntegerParameterType name="TYPE" signed="false">
+				<xtce:IntegerDataEncoding sizeInBits="1" encoding="unsigned" />
+			</xtce:IntegerParameterType>
+			<xtce:IntegerParameterType name="SEC_HDR_FLG" signed="false">
+				<xtce:IntegerDataEncoding sizeInBits="1" encoding="unsigned" />
+			</xtce:IntegerParameterType>
+			<xtce:IntegerParameterType name="PKT_APID" signed="false">
+				<xtce:IntegerDataEncoding sizeInBits="11" encoding="unsigned" />
+			</xtce:IntegerParameterType>
+			<xtce:IntegerParameterType name="SEQ_FLGS" signed="false">
+				<xtce:IntegerDataEncoding sizeInBits="2" encoding="unsigned" />
+			</xtce:IntegerParameterType>
+			<xtce:IntegerParameterType name="SRC_SEQ_CTR" signed="false">
+				<xtce:IntegerDataEncoding sizeInBits="14" encoding="unsigned" />
+			</xtce:IntegerParameterType>
+			<xtce:IntegerParameterType name="PKT_LEN" signed="false">
+				<xtce:IntegerDataEncoding sizeInBits="16" encoding="unsigned" />
+			</xtce:IntegerParameterType>
+			<xtce:IntegerParameterType name="TEST_PACKET.SHCOARSE" signed="false">
```

Comment:
I'm not sure if you have a better way to handle this, but we have potentially duplicate names across packets with different lengths/datatypes. For example, someone may want to say packet1 has "VAR1 == uint3", but packet2 has "VAR1 == uint8" (especially relevant for spare/fill that are different beetween packets). So, how do we represent that in the XTCE? On IMAP we chose to prepend the `packetName.mnemonic` to avoid this, but it does add a decent amount of bloat and then does not allow someone to directly go in and say `packet["mnemonic"]`, rather they have to do `packet["packet_name.mnemonic"]`. We strip that off ourselves, which works OK for us. But it would be good to hear if there is something we are missing or could be doing better.